src/toolkits/drawing_classifier/data_preparation.cpp

```diff
@@ -448,9 +448,10 @@ gl_sframe _drawing_classifier_prepare_data(const gl_sframe &data,
     for (const auto& row: data.range_iterator()) {
         const flexible_type &strokes = row[column_index_map[feature]];
         flex_list current_stroke_based_drawing = strokes.to<flex_list>();
-        bitmaps.push_back(convert_stroke_based_drawing_to_bitmap(
-            current_stroke_based_drawing, row_number)
-        );
+        flex_image bitmap = convert_stroke_based_drawing_to_bitmap(
+            current_stroke_based_drawing, row_number);
+        bitmaps.push_back(bitmap);
+        std::cout << row_number;
```

Comment:
might consider suppressing the print.

Reply:
Let's just revert this file. Consider staging debugging changes under a separate commit that you don't push to GH for review

Reply:
Thanks.